docs/documentation/components/tabs.html

```diff
@@ -16,60 +16,78 @@
 
 {% capture tabs_example %}
 <div class="tabs">
-  <ul>
-    <li class="is-active"><a>Pictures</a></li>
-    <li><a>Music</a></li>
-    <li><a>Videos</a></li>
-    <li><a>Documents</a></li>
+  <ul role="tab">
+    <li role="presentation" class="is-active">
+      <a href="#picture-panel" id="picture-tab"
```

Comment:
Using Links vs Buttons?
- by using links will make the components work well if there's no JS loaded.